doc/conda/Jenkinsfile.conda.template

```diff
@@ -0,0 +1,92 @@
+pipeline {
+    agent any
+    environment {
+        package_name = "{csc_name}"
+        dockerImageName = "lsstts/conda_package_builder:latest"
+        container_name = "conda_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+    }
+
+    stages {
+        stage("Pull Docker Image") {
+            steps {
+                script {
+                sh """
+                docker pull ${dockerImageName}
```

Comment:
My preference would be to use docker plugin instead of running docker from shell. Let me know if you need a help.

Reply:
Thanks for the comment @pkubanek ... Is this a good example on how to use the docker plugin? https://github.com/lsst-ts/ts_salobj/blob/u/pkubanek/jenkinsfile-conda-node/Jenkinsfile.conda